src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -155,169 +155,166 @@ const FooterExtraActions = styled.div`
   }
 `
 
-export const Sidebar = React.forwardRef<HTMLDivElement, Props>(
-  ({ sticky, offerData, refetch, onCheckoutOpen }, ref) => {
-    const textKeys = useTextKeys()
-    const market = useMarket()
-    const [
-      discountCodeModalIsOpen,
-      setDiscountCodeModalIsOpen,
-    ] = React.useState(false)
-    const [isSidebarVisible, setIsSidebarVisible] = React.useState(true)
+export const Sidebar = React.forwardRef<HTMLDivElement, Props>(function sidebar(
```

Comment:
I kinda dislike this rule and would rather add `'react/display-name': 'off',` what do you think? @styrelius @palmenhq

Reply:
Sure!

Reply:
I have no opinion whatsoever in this matter so I can definitely turn that rule off and change those now named functions back into anonymous functions!